src/analysis/retail/priest/discipline/modules/spells/Evangelism.tsx

```diff
@@ -86,6 +97,15 @@ class Evangelism extends Analyzer {
         position={STATISTIC_ORDER.CORE(0)}
         size="flexible"
         category={STATISTIC_CATEGORY.TALENTS}
+        tooltip={
+          <>
+            <SpellLink spell={TALENTS_PRIEST.EVANGELISM_TALENT} /> Active Healing:{' '}
+            {formatNumber(this.evangelismActiveHealing)}
+            <br />
+            <SpellLink spell={SPELLS.ATONEMENT_HEAL_NON_CRIT} /> extension healing:{' '}
+            {formatNumber(evangelismStatistics.reduce((p, c) => p + c.healing, 0))}
```

Comment:
You should put the `evangelismStatistics.reduce((p, c) => p + c.healing, 0)` into a getter since you're using it in multiple places